app/views/shared/views/ems_common/_show.html.haml

```diff
@@ -12,5 +12,7 @@
       var miq_after_onload = "miqAsyncAjax('#{url_for(:action => @ajax_action, :id => @record)}');"
   - elsif @showtype == "dialog_provision"
     = render :partial => "shared/dialogs/dialog_provision"
+  - elsif @display == "main"
+    = render :partial => @showtype
```

Comment:
@simon3z i am not sure why you need to add render :partial => @showtype, we do have else branch in this view app/views/shared/views/ems_common/_show.html.haml

``- else
     = render :partial => "shared/views/ems_common/#{@showtype}"``